### PR TITLE
Fixes errors caused by viewing unwatched topics of non owners profile

### PR DESCRIPTION
**Pull request by Butterflysaway**
Guest
 Today at 12:54:08 PM
 removed
 removed
Backtrace information Type of error: Undefined
8: Undefined index: is_topics
 https://website.com/index.php?action=profile;area=showposts;sa=unwatchedtopics;u=2056015
 /var/www/removed/public_html/Themes/default/Profile.template.php (Line 544)

 Guest
 Today at 12:54:08 PM
 removed
 removed
Backtrace information Type of error: General
2: Invalid argument supplied for foreach()
 https://website.com/index.php?action=profile;area=showposts;sa=unwatchedtopics;u=2056015
 /var/www/removed/public_html/Themes/default/Profile.template.php (Line 505)

 Guest
 Today at 12:54:08 PM
 removed
 removed
Backtrace information Type of error: Undefined
8: Undefined index: posts
 https://website.com/index.php?action=profile;area=showposts;sa=unwatchedtopics;u=2056015
 /var/www/removed/public_html/Themes/default/Profile.template.php (Line 505)

Signed-off-by: Butterflys Away <68415596+Butterflysaway@users.noreply.github.com>

### CODE REVIEW

File: Sources/Profile-View.php [hunk 1]
```diff
@@ -833,10 +833,14 @@ function showPosts($memID)
 	// Shortcut used to determine which $txt['show*'] string to use for the title, based on the SA
 	$title = array(
 		'attach' => 'Attachments',
-		'unwatchedtopics' => 'Unwatched',
 		'topics' => 'Topics'
```
Comment (SychO9) [inline Sources/Profile-View.php:836]: sorry this comment is unrelated to the pr but, is it me or are these hardcoded strings ?

Comment (Butterflysaway) [reply Sources/Profile-View.php:836]: Yes they are

Comment (frandominguezl) [reply Sources/Profile-View.php:836]: 'Attachments' can be replaced for $txt['showAttachments'], 'Topics' can be replaced for $txt['showTopics'] and 'Unwatched' for $txt['showUnwatched'].

Comment (SychO9) [reply Sources/Profile-View.php:836]: Oh I see it now, indeed. They are not hardcoded strings

**Review by MissAllSunday — APPROVED** [review]

**Comment by MissAllSunday** [comment]
Hey @Butterflysaway Do you have an SMF.org account?

**Pull request merged**